### ISSUE TITLE
Feature request - support for RottenTomatoes

### ISSUE THREAD

**Issue by Iristyle** [roboticsound/Pulsarr#36]
RT support would be awesome. Thanks!

**Comment by roboticsound** [comment]
I would love to implement this but as yet do not really have any ideas how to. I'm open to suggestions.

**Comment by piwel** [comment]
In the same request, I'd like to ask for allocine.fr

Maybe is it possible to be inspired by what was done on the Couchpotato plugin : 
- [for Allocine](https://github.com/CouchPotato/CouchPotatoServer/blob/master/couchpotato/core/media/movie/providers/userscript/allocine.py)
- [for RottenTomatoes](https://github.com/CouchPotato/CouchPotatoServer/blob/master/couchpotato/core/media/movie/providers/userscript/rottentomatoes.py)
- [for a few others…](https://github.com/CouchPotato/CouchPotatoServer/tree/master/couchpotato/core/media/movie/providers/userscript)

**Comment by roboticsound** [comment]
Yeah I had been thinking that this may be the way to go, I'm just worried about get false positives when searching by title & year rather than by ID.

**Comment by intelligence** [comment]
Is this live? Can't get it to work on RT.

**Comment by mariotacke** [comment]
Same here. RT went through a redesign to React, maybe it broke the layout when parsing?

**Comment by zanetaylor** [comment]
This seems to be working in the current repo version, but not in the version that's currently published to the Firefox and Chrome extension libraries, which hasn't been updated since February 21.

**Comment by piejanssens** [comment]
You can close this issue.
